lib/objectBuilder.js

```diff
@@ -431,13 +431,13 @@ module.exports = function () {
   function makeWizard() {
     var rec = _.assign(_createObject(), deviceInfo, {
       type: 'wizard',
-      recommended: REQUIRED,
+      recommended: OPTIONAL,
```

Comment:
Good idea. I have just tested this by uploading MedT Direct via platform with those fields disabled and it works as expected.